src/zope/app/apidoc/static.py

```
@@ -256,316 +314,297 @@ def sendMessage(self, msg, verbosity=4):
-""")
-
-retrieval.add_option(
-    '--password', '--pwd', action="store", dest='password',
-    help="""\
-Password to access the Web site.
-""")
-
-retrieval.add_option(
-    '--add', '-a', action="append", dest='additional_urls',
-    help="""\
-Add an additional URL to the list of URLs to retrieve. Specifying those is
-sometimes necessary, if the links are hidden in cryptic JAvascript code.
-""")
-
-retrieval.add_option(
-    '--ignore', '-i', action="append", dest='ignore_modules',
-    help="""\
-Add modules that should be ignored during retrieval. That allows you to limit
-the scope of the generated API documentation.
-""")
-
-retrieval.add_option(
-    '--load-all', '-l', action="store_true", dest='import_unknown_modules',
-    help="""\
-Retrieve all referenced modules, even if they have not been imported during
-the startup process.
-""")
-
-parser.add_option_group(retrieval)
+        filepath = self._handleDirForResponse(link)
 
-######################################################################
-# Reporting
+        contents = self._handleFindLinksForResponse(link)
 
-reporting = optparse.OptionGroup(
-    parser, "Reporting", "Options that configure the user output information.")
+        # Write the data into the file
+        if not isinstance(contents, bytes):
+            contents = contents.encode('utf-8')
+        try:
+            with open(filepath, 'wb') as f:
+                f.write(contents)
+        except IOError: # pragma: no cover
+            # The file already exists, so it is a duplicate and a bad one,
+            # since the URL misses `index.hml`. ReST can produce strange URLs
+            # that produce this problem, and we have little control over it.
 
-reporting.add_option(
-    '--verbosity', '-v', type="int", dest='verbosity',
-    help="""\
-Specifies the reporting detail level.
-""")
+            # XXX: That sounds wrong. We don't specify to open the file in
+            # exclusive creation.
```

(Or the user could be running out of disk space -- I'd print a warning to stderr instead of just ignoring the error.  Not part of this PR.)